### ISSUE TITLE
Brazilian Portuguese translation

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#321]
```
I'm working on a Brazilian Portuguese translation for Andor's Trail. Translated 
already everything, except for the conversation list, where I finished about 
50% of the dialogs.

For the files already present at the Portugal's translation, I just corrected 
the language.

For the conversation (and part of the questlist file), I used a small perl 
script that extracted all strings and stored them into a file. Then, I used 
Google translator, and manually reviewed the results and used a second script 
to store them back again into the files. That warranted that nothing got 
mangled, and helped to get a translation that would otherwise take much more 
time. I'm enclosing both perl scripts.

I'm pretty sure that there are still some ugly translations there, and lots of 
space for improvements, but that's a way better for Brazilians to read than 
before, especially for those that aren't so familiar with English, like my 
kids;)
```

Original issue reported on code.google.com by `mauroche...@gmail.com` on 13 Oct 2012 at 8:47

Attachments:
- [0001-Copy-Portuguese-Pt-translation-to-Brazilian-Pt-value.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0001-Copy-Portuguese-Pt-translation-to-Brazilian-Pt-value.patch)
- [0002-Translate-strings.xml-to-pt-BR.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0002-Translate-strings.xml-to-pt-BR.patch)
- [0003-Add-0.6.11-conditions-at-pt-BR-content_actorconditio.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0003-Add-0.6.11-conditions-at-pt-BR-content_actorconditio.patch)
- [0004-Translate-strings_about.xml-to-pt-BR.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0004-Translate-strings_about.xml-to-pt-BR.patch)
- [0005-Translate-content_questlist.xml-to-Brazilian-Portugu.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0005-Translate-content_questlist.xml-to-Brazilian-Portugu.patch)
- [0006-Convert-content_itemlist.xml-to-Brazilian-Portuguese.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0006-Convert-content_itemlist.xml-to-Brazilian-Portuguese.patch)
- [0007-Convert-about-half-of-the-dialogs-of-content_convers.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0007-Convert-about-half-of-the-dialogs-of-content_convers.patch)
- [0008-Add-credits-to-Brazilian-Portuguese-translation.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/0008-Add-credits-to-Brazilian-Portuguese-translation.patch)
- [get_strings.pl](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/get_strings.pl)
- [translation.pl](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-0/translation.pl)

**Comment by GoogleCodeExporter** [comment]
```
The Portugal's translation were done on 0.6.10. Due to that, newer monsters are 
missing. Add the new monsters to the Brazilian translation (and fix one typo).
```

Original comment by `mauroche...@gmail.com` on 13 Oct 2012 at 11:38

Attachments:
- [0009-Brazilian-Portuguese-translation-for-monsterlist-sin.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-1/0009-Brazilian-Portuguese-translation-for-monsterlist-sin.patch)

**Comment by GoogleCodeExporter** [comment]
```
Add more dialog translations. With that, 67% of the conversation got already 
translated to Brazilian Portuguese (2911 of 4344).
```

Original comment by `mauroche...@gmail.com` on 16 Oct 2012 at 2:41

Attachments:
- [0010-Brazilian-Portuguese-translation-add-more-dialogs.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-2/0010-Brazilian-Portuguese-translation-add-more-dialogs.patch)

**Comment by GoogleCodeExporter** [comment]
```
Finish the Brazilian Portuguese translation

The remaining patches finishes the Brazilian portuguese translation:
   - Add the remaining 33% of the conversationlist;
   - Fixes some issues at the questlist;
   - Add Brazilian Portuguese to AndorsTrailEdit translations.html;
   - Translate a few more monsters that I missed.

All translations got validated by AndorsTrailEdit translations.html. The only 
warnings are due to "Kazul's language dialogs" and proper names at the monster 
lists.
```

Original comment by `mauroche...@gmail.com` on 21 Oct 2012 at 2:24

Attachments:
- [0011-Finish-the-conversation-translation-to-Brazilian-Por.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-3/0011-Finish-the-conversation-translation-to-Brazilian-Por.patch)
- [0012-Fixe-some-issues-at-the-questlist-at-the-Portuguese-.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-3/0012-Fixe-some-issues-at-the-questlist-at-the-Portuguese-.patch)
- [0013-Add-Brazilian-Portuguese-to-AndorsTrailTranslations..patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-3/0013-Add-Brazilian-Portuguese-to-AndorsTrailTranslations..patch)
- [0014-Translate-a-few-more-monsters.patch](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-321/comment-3/0014-Translate-a-few-more-monsters.patch)

**Comment by GoogleCodeExporter** [comment]
```
Thanks yet again, mchehab! 
Merged in f3e044028f8b8abc8968ca62fbdd88f83597e4a9 and 
e4fa988d1171c7819f3657a95bf063a972e00201. Marking this as ready for v0.6.12.

Great work!
```

Original comment by `oskar.wi...@gmail.com` on 22 Oct 2012 at 6:17
- Changed state: **Ready**
- Added labels: **Type-Translation**, **Milestone-0.6.12**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:44
- Changed state: **Fixed**
